src/Rules/Library/BoundingRectangleCompletelyObscuresContainer.cs

```diff
@@ -34,7 +34,10 @@ protected override Condition CreateCondition()
         {
             // Windows can be any size, regardless of their parents
```

Comment:
This comment now feels "detached" from the check. Should it be closer to the return statement, and possibly updated to reference dialogs?

Reply:
Updated the comment.